packages/web-components/src/components/masthead/megamenu-top-nav-menu.ts

```diff
@@ -134,6 +134,14 @@ class DDSMegaMenuTopNavMenu extends DDSTopNavMenu {
         forEach(doc.querySelectorAll((this.constructor as typeof DDSMegaMenuTopNavMenu).selectorOverlay), item => {
           (item as DDSMegaMenuOverlay).active = this.expanded;
         });
+
+        import(`./megamenu-left-navigation`);
+        import('./megamenu-category-link');
+        import('./megamenu-category-group');
+        import('./megamenu-category-group-copy');
+        import('./megamenu-link-with-icon');
+        import('./megamenu-overlay');
```

Comment:
We'll probably need to add a way to track once these have been imported, so we don't try to re-import everytime we update the 'expanded' property on a top-nav-menu item

Reply:
Good idea, I added a boolean check before importing the components.